docs/AboutUs.md

```diff
@@ -9,6 +9,15 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
+### Yeo Jun Wei
+
+<img src="images/junwei3112.png" width="200px">
+
+[[github](https://github.com/JunWei3112)] [[portfolio](team/yeojunwei.md)]
```

Comment:
A minor suggestion would be to have the portfolio link one line below the GitHub link, just so that it is standardised with how others have written it. Otherwise LGTM.

Reply:
Yes, I agree with your suggestion and have amended it. Thanks!